src/main/java/uk/gov/hmcts/reform/hmc/data/HearingEntity.java

```diff
@@ -110,6 +111,32 @@ public Optional<HearingResponseEntity> getLatestHearingResponse() {
             : Optional.empty();
     }
 
+    public String getHearingStatus() {
```

Comment:
Just thinking, maybe we should name this a bit more explicitly to make it stand out more compared to the existing `getStatus()`?
```suggestion
    public String getDerivedHearingStatus() {
```

Reply:
Updated in: [c279ba1](https://github.com/hmcts/hmc-cft-hearing-service/pull/168/commits/c279ba1dcddc21323803f77eb741d161464b5de9)